conformance/base/manifests.yaml

```diff
@@ -0,0 +1,204 @@
+# This file contains the base resources that most conformance tests will rely
+# on. This includes 4 namespaces along with Server and Client Deployments in
+# each of them that can be used as backends and probes for testing traffic.
+# Create 4 namespaces
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: network-policy-conformance-gryffindor
+  labels:
+    conformance-house: gryffindor
```

Comment:
LOL 👍

Reply:
hehe yeah I thought its easier to discuss and picture the tests in our head if we run into bugs like "harry from gryffindor couldn't talk to cedric from hufflepuff because of policyA"